src/main/java/com/qouteall/immersive_portals/McHelper.java

```diff
@@ -290,12 +291,15 @@ public static void transformationPush(MatrixStack matrixStack) {
     
     public static List<GlobalTrackedPortal> getGlobalPortals(World world) {
         List<GlobalTrackedPortal> result;
-        if (world.isClient) {
+        if (world instanceof ClientWorld) {
```

Comment:
The usage of `ClientWorld` may cause ClassNotFoundException in dedicated server. It can be moved to `CHelper#getClientGlobalPortal`

Reply:
Ahh, right. Whoops